### PR TITLE
Make the summary page submit button text customisable

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Fixes an issue where the `Submit` button on the Summary page was not picking up custom content from schemas using `content.submit_button`

### How to review 
Use the `test_summary_with_submission_text` schema to ensure the submit button content matches the content set in the schema.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[CE & Individual - submit button says 'Submit answers' when should say 'Submit census'](https://trello.com/c/FL2WBbJc/4285-ce-submit-button-says-submit-answers-when-should-say-submit-census)

**Review by petechd — CHANGES_REQUESTED** [review]
I think summary page title change is still missing (templates/summary.html). Should be "Submit census - Census 2021" not "Summary - Census 2021".

**Review by petechd — APPROVED** [review]

File: templates/summary.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% import 'macros/helpers.html' as helpers %}
 
-{% set summary_page_title = _("Summary") %}
+{% set summary_page_title = _("Submit census") %}
```
Comment (MebinAbraham) [inline templates/summary.html:6]: This should come from the context.
```suggestion
{% set summary_page_title = content.title %}
```

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

File: tests/integration/questionnaire/test_questionnaire_page_titles.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def test_should_have_question_in_page_title_when_loading_summary(self):
         # When
         self.post({"answer": ""})
         # Then
-        self.assertEqualPageTitle("Summary - Census 2021")
+        self.assertEqualPageTitle("Submit census - Census 2021")
```
Comment (MebinAbraham) [inline tests/integration/questionnaire/test_questionnaire_page_titles.py:34]:
```suggestion
        self.assertEqualPageTitle("Check your answers and submit - Census 2021")
```

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**